PowerEditor/src/NppIO.cpp

```diff
@@ -2399,9 +2399,36 @@ bool Notepad_plus::loadSession(Session & session, bool isSnapshotMode, const wch
 		}
 		else
 		{
-			BufferID foundBufID = MainFileManager.getBufferFromName(pFn);
-			if (foundBufID == BUFFER_INVALID)
-				lastOpened = nppGUI._keepSessionAbsentFileEntries ? MainFileManager.newPlaceholderDocument(pFn, MAIN_VIEW, userCreatedSessionName) : BUFFER_INVALID;
+			bool recoveredBackup = false;
+			//If it's in snapshot mode, there's a backupFilePath present, but the existing path doesn't exist
+			if (isSnapshotMode && !session._mainViewFiles[i]._backupFilePath.empty())
+			{
+				//Jusko was here, December 2024 <3

```

Comment:
I don't think such a comment is going to be acceptable.

Reply:
No offense, but this is just so similar to a doggy does when it marks its spots outside...  :-)

Reply:
@xomx Could you provide the link please?

Reply:
@donho 
I meaned the same thing (`//Jusko was here, December 2024 <3`) as @alankilborn pointed out.
It just seems ridiculously similar to what dogs do outside when they want to let other dogs know they have been there... :-)

Anyway, due to the extensive reformatting (right one, though) in this PR, it took me a while to find a place with any relevant changes to the code.

Reply:
No offense taken. Is this PR okay as is, or would the two of you rather I revise it without the commentary and with only relevant code changes (and not the reformatting)?

Reply:
> //Jusko was here, December 2024 <3

If I allow it, Notepad++ source code will be a graffiti wall.